skymp5-server/cpp/lib_espm/Loader.h

```diff
@@ -10,16 +10,20 @@ namespace fs = std::filesystem;
 
 #include "Combiner.h"
 #include "espm.h"
+#include "impl/IBuffer.h"
 
 namespace espm {
+
 class Loader
 {
 public:
-  class LoadError : public std::logic_error
+  class LoadError : public std::runtime_error
```

Comment:
We probably should drop this at all and just use `std::runtime_error` everywhere.

Reply:
Sounds good enough, got rid of `LoadError`

Reply:
Also dropped excess checks, as `std::filesystem::file_size` throws, if file doesn't exist